magic.h

```diff
@@ -56,6 +56,10 @@
 
 #endif /* ! __APPLE__ */
 
+#ifndef REG_STARTEND /* musl doesn't define this */
+#define REG_STARTEND (1 << 2) /* from glibc */
```

Comment:
If musl's regexec doesn't implement REG_STARTEND, which is not clear from your pull request, it does not seem correct to pass in some value gleaned from glibc. How does the behaviour change, is it simply ignored, or does it enable compatibility with glibc/*BSD without defining REG_STARTEND?

If someone is serious in supporting Linux w/ musl, they need to do the work evaluating what is required here, does this make sense in both places regexec(3) is used? I'm also curious about the seccomp-sandboxing support, did the policy as written for glibc work for musl, or are you disabling it?